# Request
---

# Response
uint32 count